### PR TITLE
Fix to markdown keeping table from being displayed

### PR DESCRIPTION
**Pull request by satellite-no**
Line 96 should be a space, I suspect this is keeping Github from displaying this table correctly.

### CODE REVIEW

**Comment by satellite-no** [comment]
Reference page that is displaying incorrectly.
https://splunk.github.io/splunk-operator/CustomResources.html#common-spec-parameters-for-splunk-enterprise-resources

**Comment by satellite-no** [comment]
Can I get a review.. this but in the documentation makes it extremely difficult to read CRD specs.
@gaurav-splunk  @vivekr-splunk @smohan-splunk

**Comment by gaurav-splunk** [comment]
> Can I get a review.. this bug in the documentation makes it extremely difficult to read CRD specs. @gaurav-splunk @vivekr-splunk @smohan-splunk

Added few reviewers, will review it soon.

**Comment by gaurav-splunk** [comment]
I changed the target branch, so, please resolve all the merge conflicts

**Comment by sgontla** [comment]
Thanks for addressing this change. Seems like the issue is specific to the github.io, and it looks good on the github.com. Sounds, interesting.

**Review by sgontla — APPROVED** [review]

**Review by gaurav-splunk — APPROVED** [review]

File: docs/CustomResources.md [hunk 1]
```diff
@@ -139,7 +139,8 @@ Enterprise resources, including: `Standalone`, `LicenseManager`,
 | readinessInitialDelaySeconds | readinessProbe [initialDelaySeconds](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes) | Defines `initialDelaySeconds` for Readiness probe
 | livenessInitialDelaySeconds | livenessProbe [initialDelaySeconds](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command) | Defines `initialDelaySeconds` for the Liveness probe
 | imagePullSecrets | [imagePullSecrets](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) | Config to pull images from private registry. Use in conjunction with `image` config from [common spec](#common-spec-parameters-for-all-resources)
-## LicenseManager Resource Spec Parameters
+
+## LicenseMaster Resource Spec Parameters
```
Comment (sgontla) [inline docs/CustomResources.md:143]: @satellite-no , Just curious how are you testing this fix.
Looking at the above table, i think the issue could be due to the missing pipe for the following lines(?). thoughts?

> | serviceAccount | [ServiceAccount](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) | Represents the service account used by the pods deployed by the CRD |
| extraEnv | Extra environment variables | Extra environment variables to be passed to the Splunk instance containers
| readinessInitialDelaySeconds | readinessProbe [initialDelaySeconds](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes) | Defines `initialDelaySeconds` for Readiness probe
| livenessInitialDelaySeconds | livenessProbe [initialDelaySeconds](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command) | Defines `initialDelaySeconds` for the Liveness probe
| imagePullSecrets | [imagePullSecrets](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) | Config to pull images from private registry. Use in conjunction with `image` config from [common spec](#common-spec-parameters-for-all-resources)

Comment (satellite-no) [reply docs/CustomResources.md:143]: I'm testing it with vscode markdown previewer and an online markdown preview tool (https://markdownlivepreview.com/).  Both of with actually interpret the markdown correctly so its been a bit of a guessing game :(.  I did however over look the missing pipe which I'd agree is probably the issue github.io is having with it so my added space is probably just going towards proper markdown formatting. LOL

I'll update to include the pipes

Comment (sgontla) [reply docs/CustomResources.md]: Thank you!

**Review by sgontla — APPROVED** [review]

File: docs/CustomResources.md [hunk 2]
```diff
@@ -135,11 +135,12 @@ Enterprise resources, including: `Standalone`, `LicenseManager`,
...
-## LicenseManager Resource Spec Parameters
+| extraEnv | Extra environment variables | Extra environment variables to be passed to the Splunk instance containers |
+| readinessInitialDelaySeconds | readinessProbe [initialDelaySeconds](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes) | Defines `initialDelaySeconds` for Readiness probe |
+| livenessInitialDelaySeconds | livenessProbe [initialDelaySeconds](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command) | Defines `initialDelaySeconds` for the Liveness probe |
+| imagePullSecrets | [imagePullSecrets](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) | Config to pull images from private registry. Use in conjunction with `image` config from [common spec](#common-spec-parameters-for-all-resources) |
+
+## LicenseMaster Resource Spec Parameters
```
Comment (sgontla) [inline docs/CustomResources.md:143]: nit: Sorry one last minor comment. This is LicenseManager on the latest develop, as we can see the example below.

Comment (satellite-no) [reply docs/CustomResources.md:143]: Oh good call!

**Comment by satellite-no** [comment]
@sgontla can you merge this if its good?

**Pull request merged**